terraform/eks/README.md

```diff
@@ -40,28 +47,42 @@ For details on Morpheus functionality, [click here](https://github.com/NVIDIA-AI
 - AWS CLI (configured)
 - kubectl
 - jq
-- API Keys: GitHub, NVD, SerpApi, NVIDIA Inference Microservices (NIM)
+- API Keys:
+  - GitHub - Generate [GitHub Personal Access Token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens#creating-a-personal-access-token-classic)
+  - NVD - Request a NIST National Vulnerability Database [API Key](https://nvd.nist.gov/developers/request-an-api-key)
+  - SerpApi - Generate a [Private API Key](https://serpapi.com/) - Requires Sign up / Login
+  - NVIDIA Inference Microservices (NIM) - [Generate an NGC API Key](https://docs.nvidia.com/ai-enterprise/deployment/spark-rapids-accelerator/latest/appendix-ngc.html) - Requires Sign up / Login
+
+Ensure your shell supports local command execution (Bash/Zsh) recommended, or use WSL on Windows.
 
-Ensure your shell supports local command execution (Bash/Zsh recommended, or use WSL on Windows).
+Also, make sure that Docker daemon is running in your environment.
+
+1. Clone the repository:
 
-1. Set up environment variables:
+   ```bash
+   git clone https://github.com/aws-samples/gen-ai-cve-patching.git
+   cd gen-ai-cve-patching/
+   ```
+
+2. Set up environment variables:
 
 ```bash
-export BASE_PATH=</path/to/root/project>
+export BASE_PATH=$(pwd)
 export AWS_REGION="$(aws configure get region)"
 export ACCOUNT_ID="$(aws sts get-caller-identity --query Account --output text)"
 export GITHUB_USERNAME="your_github_username"
 ```
 
 ## Quick Start
 
-1. Clone the repository:
-   ```bash
-   git clone https://github.com/aws-samples/gen-ai-cve-patching.git
-   cd gen-ai-cve-patching/terraform/eks
-   ```
+1. Navigate to the respective Terraform directory.
+
+    ```bash
+    cd terraform/eks
+    ```
 
 2. Create `terraform.tfvars` with your API keys:
+
    ```hcl
    github_token    = "your_github_token"
    github_username = "your_github_username"
```

Comment:
There are more TF vars than those, nvd_key etc

Reply:
Yes, but these are like sensitive data, and we can avoid. them to be pushed to repos. We can also instruct to pass via command line, but opens room to errors IMHO.